src/components/HoldingsTable.tsx

```diff
@@ -1,94 +1,22 @@
 import { FC } from "react"
-import { Address, useAccount } from "wagmi"
 
-import { ProductType, VaultType } from "@/lib/types"
-import { enabledNetworks } from "@/lib/wagmi"
-import {
-  useActiveChainId,
-  useClientReady,
-  useCompounderVault,
-  useHoldingsVaults,
-  useListCompounders,
-} from "@/hooks"
+import { ProductType } from "@/lib/types"
 
-import { TableDisconnected, TableEmpty, TableLoading } from "@/components/Table"
-import { VaultRow } from "@/components/VaultRow"
-import { VaultTable } from "@/components/VaultRow/lib"
+import CompounderHoldingsTable from "@/components/Compounder/CompounderHoldingsTable"
+import ConcentratorHoldingsTable from "@/components/Concentrator/ConcentratorHoldingsTable"
 
 const HoldingsTable: FC<{
```

Comment:
This component is unnecessary. Just use the desired `-HoldingsTable` component in `pages/yield/index.tsx` and `pages/yield/concentrators.tsx`.